qemu/tests/block_commit_reboot.py

```diff
@@ -0,0 +1,46 @@
+import time
+import random
+from qemu.tests import blk_commit
+
+
+class BlockCommitReboot(blk_commit.BlockCommit):
+
+    def reboot(self):
+        """
+        Reset guest with system_reset;
+        """
+        params = self.parser_test_args()
+        method = params.get("reboot_method", "system_reset")
+        return super(BlockCommitReboot, self).reboot(method=method, boot_check=False)
+
+    def action_when_start(self):
+        """
+        start pre-action in new threads;
+        """
+        super(BlockCommitReboot, self).action_when_start()
+        time.sleep(random.randint(25, 100))
```

Comment:
why do you need a sleep here?

Reply:
Yea, why do you need, especially, a random sleep? Is that a test step?

Reply:
@xiywang @luckyh @hereischen We do need a random sleep here, to perform the block job during different stage of rebooting, like emulate the real manual test scenario. But the sleep time I would suggest from 0-20, current value will be too long for rebooting finish.